.gitlab-ci.yml

```diff
@@ -56,7 +56,7 @@
 
   test py39:
     stage: test
-    image: python:3.9
```

Comment:
Python 3.11 is used in the production Dockerfile so the test should do the same.
Please add more python variants.